### PR TITLE
adding full k8s tests to airgap

### PR DESCRIPTION
**Pull request by slickwarren**
port of https://github.com/rancher/rancher/pull/45899

### CODE REVIEW

File: tests/v2/actions/provisioning/creates.go [hunk 1]
```diff
@@ -66,10 +66,6 @@ const (
 	rke1NodeCorralName                   = "rke1registerNode"
 )
 
-var (
-	updateConfig = true
```
Comment (lscalabrini01) [inline tests/v2/actions/provisioning/creates.go:70]: I have one question, why was it removed?

Comment (lscalabrini01) [inline tests/v2/actions/provisioning/creates.go:70]: I have one question, why was it removed?

**Review by floatingman — DISMISSED** [review]
LGTM

File: tests/v2/actions/provisioning/creates.go (hunk 1 above)
Comment (lscalabrini01) [reply tests/v2/actions/provisioning/creates.go:70]: May be needs a rebase

Comment (slickwarren) [reply tests/v2/actions/provisioning/creates.go]: good catch, pushed the change just now.

**Review by lscalabrini01 — APPROVED** [review]

**Review by floatingman — APPROVED** [review]
LGTM

**Review by anupama2501 — APPROVED** [review]

**Review by markusewalker — APPROVED** [review]

**Pull request merged**